styles/pronouns.cc/catppuccin.user.css

```diff
@@ -0,0 +1,238 @@
+    .toast {
+      --bs-toast-color: @text;
+      background-color: darken(@surface0, 5%);
+    }
+    .toast-header {
+      color: @text;
+      background-color: @surface0;
+    }
+     .toast-body {
+      color: @text;
+    } 
+    --bs-badge-color: @text;
+    --bs-link-color-rgb: #rgbify(@blue) [];
+    --bs-body-color: @text;
+    --bs-body-bg: @crust;
+    --bs-darkk-rgb: #rgbify(@base);
+    --bs-lightt-rgb: #rgbify(@base);
+    --bs-emphasis-color: @accent-color;
+    --bs-emphasis-color-rgb: #rgbify(@accent-color) [];
+    --bs-secondary-color: @subtext0;
+    --bs-secondary-color-rgb: @subtext0;
+    --bs-secondary-bg: @base;
+    --bs-secondary-bg-rgb: #rgbify(@base) [];
+    --bs-tertiary-color: @subtext1;
+    --bs-tertiary-color-rgb: @subtext1;
+    --bs-tertiary-bg: @surface0;
+    --bs-tertiary-bg-rgb: #rgbify(@surface0) [];
+    --bs-primary-text-emphasis: @accent-color;
+    --bs-secondary-text-emphasis: @subtext1;
+    --bs-success-text-emphasis: @green;
+    --bs-info-text-emphasis: @teal;
+    --bs-warning-text-emphasis: @yellow;
+    --bs-danger-text-emphasis: @red;
+    --bs-light-text-emphasis: @subtext0;
+    --bs-primary-bg-subtle: @mantle;
+    --bs-secondary-bg-subtle: @base;
+    --bs-primary-border-subtle: @blue;
+    --bs-secondary-border-subtle: @overlay0;
+    --bs-success-border-subtle: @green;
+    --bs-info-border-subtle: @teal;
+    --bs-warning-border-subtle: @yellow;
+    --bs-danger-border-subtle: @red;
+    --bs-light-border-subtle: @overlay1;
+    --bs-dark-border-subtle: @surface1;
+    --bs-heading-color: inherit;
+    --bs-link-color: @blue;
+    --bs-link-hover-color: @blue;
+
+    --bs-code-color: @pink;
+    --bs-highlight-color: @rosewater;
+    .list-group {
+      --bs-list-group-color: var(--bs-body-color);
+      --bs-list-group-bg: var(--bs-body-bg);
+      --bs-list-group-border-color: var(--bs-border-color);
+      --bs-list-group-border-width: var(--bs-border-width);
+      --bs-list-group-border-radius: var(--bs-border-radius);
+      --bs-list-group-item-padding-x: 1rem;
+      --bs-list-group-item-padding-y: 0.5rem;
+      --bs-list-group-action-color: var(--bs-secondary-color);
+      --bs-list-group-action-hover-color: var(--bs-emphasis-color);
+      --bs-list-group-action-hover-bg: var(--bs-tertiary-bg);
+      --bs-list-group-action-active-color: var(--bs-body-color);
+      --bs-list-group-action-active-bg: var(--bs-secondary-bg);
+      --bs-list-group-disabled-color: var(--bs-secondary-color);
+      --bs-list-group-disabled-bg: var(--bs-body-bg);
+      --bs-list-group-active-color: @base;
+      --bs-list-group-active-bg: @accent-color;
+      --bs-list-group-active-border-color: @accent-color;
+      border-radius: var(--bs-list-group-border-radius);
+    }
+    // navbar
+    .navbar {
+      --bs-navbar-color: fade(@text, 55%) !important;
+      --bs-navbar-hover-color: fade(@text, 75%) !important;
+      --bs-navbar-disabled-color: fade(@text, 25%) !important;
```

Comment:
We shouldn't be fading text colors. Use subtext1, subtext0, or even overlay2 if you have to.

Reply:
I'm pretty confused by [`c1f34cb` (#1140)](https://github.com/catppuccin/userstyles/pull/1140/commits/c1f34cb80100a52c70280549b967c06e5614638b)...  it doesn't seem to address my comment and I would not expect it to look good.

Reply:
ah mb i was using darken because it seemed to be the correct one and i was wrong. The correct would be to use `rgba` with the correct opacity. ill make the changes rn

Reply:
I'm confused. How would you use "use rgba with the correct opacity"? Do you mean to revert back to `fade`? I'm saying just use `@subtext0` instead of `fade(@text, 70%)` or whatever.

Reply:
The original code for ` --bs-navbar-hover-color`:
```css
 --bs-navbar-color:rgba(var(--bs-emphasis-color-rgb), .65); 
 ```
 instead of using fade i would just rather but the rgbified color instead.

Reply:
...why?

Reply:
Because then it’s the correct color?

Reply:
You are still changing to the same opacity of the same color. I'm saying use the other, existing shades of colors in the palette. Like subtext1, subtext0, overlay2, etc.

Reply:
Im not changing the colors because on the site it just uses that variable so im pretty much just following the sites css :)